addons/im_livechat/static/src/legacy/widgets/thread.js

```diff
@@ -130,11 +130,11 @@ var ThreadWidget = Widget.extend({
                                                     this._enabledOptions;
 
         // attachments ordered by messages order (increasing ID)
-        this.attachments = _.uniq(_.flatten(_.map(messages, function (message) {
+        this.attachments = new Set(_.flatten(messages.map(function (message) {
```

Comment:
I think `_.uniq` returns an Array, not a `Set`, so to play safe, make an array from set using `[ ...newSet() ]`